### ISSUE TITLE
Requirements doc should explicitly state: WCAG will include normative guidance for authors, but not user-agents or authoring tools

### ISSUE THREAD

**Issue by spanchang** [w3c/wcag3#592]
Section 1.2 WCAG Scope states:
"The current project design does not intend to write separate specifications or normative requirements for the technology stack beyond digital content".
The above statement  follows a list that includes:
"authoring tools, such as content management systems;
user agents, such as browsers and media players;
assistive technologies, such as screen readers, screen magnifiers, and assistants for memory, organization, or simplification;"

Admittedly later on, 4.8 Scope does refer only to "informative guidance":
"The scope of the associated informative materials will include guidance for a diverse group of stakeholders including content creators, browsers, authoring tools, assistive technologies, and more".

In the interests of plain, unambiguous and understandable  text, maybe   it will be helpful to explicitly include the statement from explanation you provided in issue #224 (https://github.com/w3c/silver/issues/224).
The statement I refer to is:
"AG can produce normative guidance for authors, but not user-agents or authoring tools". 
or
"WCAG 3 is not intended to normatively cover "features an authoring tool / browser /OS should offer to facilitate / enhance accessibility" 

This clarification IMO should be right after the statement in the Requirements doc Section 1.2 WCAG Scope:
"The current project design does not intend to write separate specifications or normative requirements for the technology stack beyond digital content".
Thanks,
Sailesh
